index.html

```diff
@@ -1085,6 +924,213 @@ <h3>Implicit Release of Pointer Capture</h3>
+            to reduce perceived latency, and then discarding these predicted points once the actual points
+            are received.</p>
+
+            <figure id="figure_predicted">
+                <img src="predicted-points.png" alt="A line drawn using coalesced points, showing predicted future points">
+                <figcaption>Example of a line in a drawing application (the result of a drawing gesture from the bottom left to the top right), using the coalesced coordinates from <code>pointermove</code> events, showing the user agent's predicted future points (the grey circles).</figcaption>
+            </figure>
+
+            <p>A <a>PointerEvent</a> has an associated <dfn>predicted event list</dfn> (a list of zero or more
+            <code>PointerEvents</code>). If this event is a <code>pointermove</code> event, it is a sequence of
+            <code>PointerEvents</code> that the user agent predicts will follow the events in the
+            <a>coalesced event list</a> in the future; otherwise it is an empty list.</p>
+            <p>The number of events in the list and how far they are from the current timestamp are determined by
+            the user agent and the prediction algorithm it uses.</p>
+                
+            <p>The events in the predicted event list will have increasing
+            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>([[!WHATWG-DOM]]),
+            so the first event will have the smallest <code>timeStamp</code> which should still be greater than
+            the last event in the <a>coalesced event list</a>.</p>
+
+            <pre id="example_11" class="example" title="Conceptual approach to drawing using coalesced events and predicted events">
+                <code>
+                var predicted_points = [];
+                window.addEventListener("pointermove", function(event) {
+                    // Clear the previously drawn predicted points.
+                    for (let e of predicted_points.reverse()) {
+                        clearPoint(e.pageX, e.pageY);
+                    }
+
+                    // Draw the actual movements that happened since the last received event.
+                    for (let e of event.getCoalescedEvents()) {
+                        drawPoint(e.pageX, e.pageY);
+                    }
+                
+                    // Draw the current predicted points to reduce the perception of latency.
+                    predicted_points = event.getPredictedEvents();
+                    for (let e of predicted_points) {
+                        drawPoint(e.pageX, e.pageY);
+                    }
+                });
+                </code>
+            </pre>
+        </section>
+        
+        <section>
+            <h2><dfn>Populating the coalesced and predicted event lists</dfn></h2>
```

Comment:
"Maintaining"?  Not sure!

Reply:
While a bit awkward, I tried to combine this ... "Populating and maintaining ...". Think that's workable...